app/components/MovieContainer.js

```diff
@@ -1,31 +1,33 @@
-var React = require('react');
-var MovieElement = require('./MovieElement');
+import React from 'react';
+import Axios from 'axios';
+import MovieElement from './MovieElement';
 
 class MovieContainer extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      movies: []
-    }
+      movies: [],
+    };
   }
 
   componentDidMount() {
     Axios.get('http://varnatrd.tech/api/series').then(({ data }) => {
```

Comment:
You should add a default value to `data` so that if the request break, the code doesn't
`{ data = [] }`